paper/myst.yml

```diff
@@ -2,48 +2,56 @@
 version: 1
 project:
   id: ubcgif-2020-heagy-etal-exploration-geophysics
-  # title: TITLE OF PAPER
-  # description: SHORT DESCRIPTION HERE
+  title: Open-source software for simulations and inversions of airborne electromagnetic data
+  description: overview of the software in its application to airborne electromagnetics and demonstrate its use for visualising fields and fluxes in a forward simulation, as well as its flexibility in formulating and solving the inverse problem.
   authors:
-    # - name: AUTHOR 1
-    #   email: EMAIL
-    #   corresponding: true
-    #   orcid: ORCID
-    #   affiliation:
-    #     id: ubc
-    #     name: Department of Earth, Ocean and Atmospheric Sciences, University of British Columbia, Vancouver, British Columbia
-    # - name: Lindsey J. Heagy
-    #   email: lheagy@eoas.ubc.ca
-    #   corresponding: true
-    #   orcid: 0000-0002-1551-5926
-    #   affiliation: ubc
-  # doi: DOI
-  # arxiv: ARXIV
-  # github: GITHUB-URL
-  # date: YYYY/MM/DD # date of publication
+    - name: Lindsey J. Heagy
+      email: lheagy@eoas.ubc.ca
+      corresponding: true
+      orcid: 0000-0002-1551-5926
+      affiliation:
+        id: ubc
+        name: Department of Earth, Ocean and Atmospheric Sciences, University of British Columbia, Vancouver, British Columbia
+    - name: Seogi Kang
+      email: sgkang09@stanford.edu
+      orcid: 0000-0002-9963-936X
+      affiliation: ubc
+    - name: Rowan Cockett
+      email: rowan@curvenote.com 
+      orcid: 0000-0002-7859-8394
+      affiliation: ubc
+    - name: Douglas W. Oldenburg
+      email: doug@eos.ubc.ca
+      orcid: 0000-0002-4327-2124
+      affiliation: ubc
+  doi: 10.1080/08123985.2019.1583538
+  arxiv: 1902.08238
+  github: https://github.com/simpeg-research/heagy-2018-AEM
+  date: 2019/06/10 # date of publication
   # license: CC-BY-4.0  # uncomment this and the line below if open access
   # open_access: true
   venue:
-    # title: JOURNAL NAME
-    # url: JOURNAL URL
+    title: Exploration Geophysics
+    url: https://www.tandfonline.com/doi/full/10.1080/08123985.2019.1583538
   history:
-    # received: YYYY/MM/DD
-    # accepted: YYYY/MM/DD
-    # published: YYYY/MM/DD
+    received: 2018/10/01
+    accepted: 2019/01/25
+    published: 2019/06/10
   keywords:
-    # - KEYWORD 1
-    # - KEYWORD 2
-    # - KEYWORD 3
-    # - KEYWORD 4
+    - Airborne Electromagnetics
+    - Electromagnetic Geophysics
+    - Inversion
+    - Programming
+    - 3D Modelling
   abbreviations:
-    # CSEM: controlled source electromagnetics
-    # EM: electromagnetics
+    TDEM: Time Domain Electromagnetic
+    EM: electromagnetics
   thumbnail: thumbnail.png
-  # numbering:
-  #   headings: true
+  numbering:
+    headings: true
 site:
   template: article-theme
   actions:
     - title: pdf
-      # url: NAME-OF-PDF.pdf
+      url: Open-source software for simulations and inversions of airborne electromagnetic data.pdf
```

Comment:
would you mind removing spaces in the filepath here? That can make things complicated, so spaces can be replaced with dashes or underscores. We can also rename the pdf to simplify it `Heagy-etal-2020-exploration-geophysics.pdf`

Reply:
Actually, looking at the repository, the name of the pdf is already `heagy-etal-2020-exploration-geophysics.pdf` , so if you could please just change the text here, that would be great 

![image](https://github.com/ubcgif/2020-heagy-etal-exploration-geophysics-aem/assets/6361812/9026c939-1039-4691-afde-c0998e2bd00d)